### PR TITLE
adding php-version

### PR DESCRIPTION
**Pull request by sroehrl**
Adding PHP (and myself) to the list

### CODE REVIEW

**Comment by RaidAndFade** [comment]
Your pull request seems good, but must be checked by a human maintainer for the following reason(s):
1. Modifications to contributors.md must be checked manually, thanks for your submission.
2. These submissions must be checked manually because of the potential complexity of submissions, if you've reached this point in the process you're most likely a valid submission.

Please be patient until a maintainer has time to look over your contribution.

***This is an automated response, beep boop.
if you think there is a mistake, contact a maintainer***
> 8625f5ce252d1ed66ace8d9ea50e125194a759cb

**Comment by RaidAndFade** [comment]
Your code should not sleep. I want to send yes as fast as possible!

**Comment by sroehrl** [comment]
> Your code should not sleep. I want to send yes as fast as possible!

Fair enough. Done!

**Comment by RaidAndFade** [comment]
I have added that as a requirement of the auto-grader, thank you for helping me improve it.

Thank you for your submission.

File: contributors.md [hunk 1]
```diff
@@ -19,8 +19,12 @@ _(Do not modify this list unless you are a maintainer)_
 _(People who have participated in the challenges!)_
 
 - [RaidAndFade](https://github.com/raidandfade)
+
+- [sroehrl](https://github.com/sroehrl)
```
Comment (RaidAndFade) [inline contributors.md:22]: Please remove the spacing around your link, The list will look nicer when it is condensed as we get more people.

Comment (sroehrl) [reply contributors.md:22]: I suppose you are talking about the new lines? I changed it.

**Comment by RaidAndFade** [comment]
Thank you for your submission! 😃

**Pull request merged**